src/MainWindow.vala

```diff
@@ -151,11 +162,22 @@ public class AppCenter.MainWindow : Gtk.ApplicationWindow {
         installed_view = new Views.InstalledView ();
         search_view = new Views.SearchView ();
 
+        network_alert_view = new Granite.Widgets.AlertView (_("Network is Not Available"),
+                                                            _("While disconnected from the network, installing and updating apps is impossible."),
```

Comment:
This line should read something more positive/constructive like `Connect to the internet to install apps.` instead of using the word "impossible."